### PR TITLE
[usage] Align credits and time in usage view

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

Right-aligns credits and time in the usage view

## Related Issue(s)

* https://github.com/gitpod-io/gitpod/issues/13154

## Release Notes

```release-note
NONE
```

## Werft options:

- [x] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: components/dashboard/src/components/UsageView.tsx [hunk 1]
```diff
@@ -310,7 +310,7 @@ function UsageView({ attributionId }: UsageViewProps) {
                                                     </div>
                                                     <div className="my-auto" />
                                                     <div className="flex flex-col col-span-3 my-auto">
-                                                        <span className="text-gray-400 dark:text-gray-500 truncate font-medium">
+                                                        <span className="text-gray-400 text-right dark:text-gray-500 truncate font-medium">
```
Comment (svenefftinge) [inline components/dashboard/src/components/UsageView.tsx:313]: did you also want to remove the truncate (elipses)?

Comment (easyCZ) [reply components/dashboard/src/components/UsageView.tsx:313]: I was planning to do that later as I need to run for a meeting but if it's that simple, can do :)

Comment (easyCZ) [reply components/dashboard/src/components/UsageView.tsx:313]: Done now

Comment (svenefftinge) [reply components/dashboard/src/components/UsageView.tsx:313]: the original issue complained about truncation on the time not the credits.
Screenshot 2022-09-21 at 10 12 27

Comment (easyCZ) [reply components/dashboard/src/components/UsageView.tsx:313]: Those are 2 different PRs, and 2 different Issues:
This one is https://github.com/gitpod-io/gitpod/issues/13154 which deals with alignment and truncation. There's another PR which deals with Credit rounding - https://github.com/gitpod-io/gitpod/issues/13056

Comment (svenefftinge) [reply components/dashboard/src/components/UsageView.tsx:313]: Yes, I'm talking about the truncation of the time property. See screenshot ☝️

Comment (easyCZ) [reply components/dashboard/src/components/UsageView.tsx:313]: Yes, that should be fixed in https://github.com/gitpod-io/gitpod/pull/13173/files#diff-f54b0b6b3eff92d9ca581427987a0fc42b091a046375f29e76283ec615370c99R306

Comment (svenefftinge) [reply components/dashboard/src/components/UsageView.tsx:313]: you emoved the truncate for the 'usage' not the 'time'.

**Review by svenefftinge — APPROVED** [review]
Looks good, adding hold in case you want to include the removal of 'truncate' on the time property

/hold

File: components/dashboard/src/components/UsageView.tsx (hunk 1 above)
Comment (easyCZ) [reply components/dashboard/src/components/UsageView.tsx]: I'm sorry, I don't follow what you mean. I'm removing truncation from the number of minutes that a workspace ran (as per the [issue](https://github.com/gitpod-io/gitpod/issues/13154)), not the usage record timestamp.

In the preview env, it renders as below, which should be the expected outcome
Screenshot 2022-09-21 at 17 13 04

**Comment by easyCZ** [comment]
I'm unholding this PR as the current form looks to solve the problem as required. Happy to do a follow-up.

/unhold

**Pull request merged**